geode-core/src/test/java/org/apache/geode/management/internal/api/LocatorClusterManagementServiceTest.java

```diff
@@ -313,4 +320,74 @@ public void deleteWithNoMember() throws Exception {
     assertThat(result.getMemberStatuses()).hasSize(0);
     assertThat(result.getStatusMessage()).contains("Successfully removed config for [cluster]");
   }
+
+  @Test
+  public void runOperationIsSuccessful() throws Exception {
+    OperationExecutor operation = new OperationExecutor() {
+      @Override
+      public CompletableFuture<OperationResult> run(Map<String, String> arguments,
+          OperationResult operationResult, Executor exe) {
+        final CompletableFuture<OperationResult> future = new CompletableFuture<>();
+        exe.execute(() -> {
+          operationResult.setStartTime(System.currentTimeMillis());
```

Comment:
are all implementations of OperationExecutor expected to track their start and finish times?  Couldn't this be handled by the framework...?